src/scss/_variables.scss

```diff
@@ -199,13 +199,13 @@ $brand-metallic-lightest-color: $black;
 // Custom brand color alternates that meet white-background contrast requirements.
```

Comment:
It would be helpful to have a short comment above a group of variables like this explaining what the abbreviations stand for. I'm assuming in this case the `-aw` is `alternate white`?